amplify/src/main/java/com/github/stkent/amplify/tracking/predicates/LastTimePredicate.java

```diff
@@ -40,4 +40,9 @@ public void eventTriggered(@NonNull IEvent event) {
             updateEventValue(event, currentTime);
         }
     }
+
+    @Override
+    public Long defaultValue() {
+        return 0L;
```

Comment:
Should we change this to Long.MAX_VALUE and update the WarmUp and Cooldown to respond to this value?